packages/lesmis/src/views/DashboardReportTabView.js

```diff
@@ -19,26 +19,62 @@ import {
   Tab,
   TabPanel,
 } from '../components';
-import { DEFAULT_DASHBOARD_GROUP } from '../constants';
+import { DEFAULT_DASHBOARD_GROUP, MIN_DATA_YEAR } from '../constants';
 
 const DashboardSection = styled(FlexCenter)`
   min-height: 31rem;
 `;
 
+const StyledSelect = styled(Select)`
+  margin: 0 1rem 0 0;
+  width: 8rem;
+`;
+
+const getYearOptions = () => {
+  const currentYear = new Date().getFullYear();
+  const years = [
+    {
+      label: 'All Data',
+      value: 'all',
+    },
+  ];
+  let year = MIN_DATA_YEAR;
+  while (year <= currentYear) {
```

Comment:
I wonder if we should go the other way around, so the most recent (probably most relevant) is closest to the top and to the default? Is this in Dave's design?

Reply:
The designs doesn't show the menu when it's open but I think you are spot on that the most relevant should be first.